igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/auth/AuthRequestInterceptor.groovy

```diff
@@ -65,7 +65,6 @@ class AuthRequestInterceptor implements RequestInterceptor {
         }
 
         String toString() {
-            log.debug("Including Basic digest in Authorization header")
```

Comment:
This is spewing a ton of log messages into igor for every request.

Reply:
👍